Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -186,15 +177,55 @@ protected
   Buildings.Controls.OBC.CDL.Reals.Abs abs1
     "Absolute value"
     annotation (Placement(transformation(extent={{-20,140},{0,160}})));
-  Buildings.Controls.OBC.CDL.Reals.Greater gre
+  Buildings.Controls.OBC.CDL.Reals.Greater gre(
+     final h=SymHys)
```

Comment:
There is no need to change the `h` to non-zero. As the inputs of the `gre` block, are parameters (the value will not change).

Reply:
Removed the hys.